webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
```

Comment:
- [ ] HTML Webpack plugin is not properly configured, and similarly, no `.html` is generated during compilation. Kindly ensure the Webpack config file has an HTML file configured in the Webpack config settings as a template using the HTML Webpack Plugin.